test/dynamo/test_misc.py

```diff
@@ -1285,10 +1285,10 @@ def sample_to_args(s):
             args = (s.input, *sample.args)
             return tuple(a.numpy() if isinstance(a, torch.Tensor) else a for a in args)
 
-        # The last sample from gather does not work in np.take_along_axis
+        # The last 2 samples from gather do not work in np.take_along_axis
         samples = list(
             sample_inputs_gather(None, "cpu", torch.float32, requires_grad=False)
```

Comment:
Maybe add a flag `include_0dim` instead of relying on a specific ordering of the samples?

Reply:
Fair enough. Done.